CONTRIBUTING.md

```diff
@@ -36,16 +47,18 @@ To be active in the Working Group, you can participate in any of these places. M
 place on the mailing list, but if you just want to comment on and raise issues, that's fine too.
 
 
-## Raising Issues
+### Post-IESG Process
 
-We use our [Github](https://github.com/quicwg/) issues lists to track items for discussion and
-their resolution.
+The Working Group has built consensus that is reflected in base-draft documents, which has been confirmed through the IETF Last Call and IESG review stages modulo any critical undiscovered issues. Design changes will no longer be considered unless there are severe security, deployment or implementation problems. modulo any open (or undiscovered) issues. The goal of the Post-IESG proces is to minimise changes that invalidate the accumulated consesus, which risks returning us to the pre Working Group Last Call stage of standardisation.
 
-Before filing a new issue, please consider a few things:
+In this process, all required parties will discuss each design or major editorial issue and proposed resolution (ideally based upon a Pull Request that specifies the exact changes to be made). Chairs will judge consensus, labelling the issue as `has-consensus`.
 
-* Issues should be just that; issues with our deliverables, **not proposals, questions or support requests**.
-* Please review the issues list to make sure that you aren't filing a duplicate.
-* If you're not sure how to phrase your issue, please ask on the [mailing list](https://www.ietf.org/mailman/listinfo/quic).
+### Raising Issues
```

Comment:
I think that you might want to say outright that this repo is closed for business and that new issues should be first discussed by the working group.  Then set expectations: say that we will only consider the most serious issues that might jeopardize the successful deployment of QUIC, such as serious security flaws or bugs without workarounds.